ldaptor/test/test_encoder.py

```diff
@@ -27,3 +27,10 @@ def test_unicode_object(self):
     def test_bytes_object(self):
         obj = b'bytes'
         self.assertEqual(to_bytes(obj), b'bytes')
+
+
+class WireStrAliasTests(unittest.TestCase):
+
+    def test_not_implemented(self):
```

Comment:
it would be nice to have a docstring describing this test scenario.
What do you try to test here.

There are no tests for the test code, so human understanding is the best we can get.

The test name and docstring should be explicit about its scope/target/purpose.

something like

```suggestion
    def test_toWire_not_implemented(self):
        """"
        toWire is an abstract method and raises `NotImplementedError`
        """
    
```

Reply:
Thanks, forgot about it! Docstrings were added